docs/writing-tests/testharness.md

```diff
@@ -199,7 +199,36 @@ dedicated worker tests and shared worker
 tests](testharness-api.html#determining-when-all-tests-are-complete), it is
 automatically invoked for tests defined using the "multi-global" pattern.
 
-## Other features of `.window.js`, `.worker.js` and `.any.js`
+## Extension tests (`.extension.js`)
```

Comment:
I think a new type of tests needs an RFC per https://github.com/web-platform-tests/rfcs

If extension tests could be regular .js tests with no special testharness support needed then my understanding is that an RFC would not be needed.

Reply:
we're working on writing an RFC for this